Cython/Utils.py

```diff
@@ -148,21 +148,31 @@ def find_root_package_dir(file_path):
 
 @cached_function
 def check_package_dir(dir, package_names):
```

Comment:
`dir` is not an ideal name since it shadows the builtin. Better change it along the way. The function below uses `dir_path`.

Reply:
Renamed dir -> dir_path.